src/aap_eda/api/serializers/eda_credential.py

```diff
@@ -222,4 +225,13 @@ def get_references(eda_credential: models.EdaCredential) -> list[dict]:
         }
         resources.append(resource)
 
+    for event_stream in used_event_streams:
+        resource = {
+            "type": "EventStream",
+            "id": event_stream.id,
+            "name": event_stream.name,
+            "url": f"api/eda/v1/event-streams/{event_stream.id}/",
```

Comment:
The value should not be hardcoded, paths are dynamic based on the settings and routing configuration. You should use `reverse` to the get the path as for example, I did [here](https://github.com/ansible/eda-server/pull/1117/files#diff-fefcb7e46a2ae907d8ac7af48e6065035f2a6017f0c427ced76f78072da75224R107). 

https://docs.djangoproject.com/en/5.1/ref/urlresolvers/


Also the name "url" might be misleading. This would not be an url strictly speaking because it doesn't contain protocol or domain. I think it would be better to say "url_path" or "uri" or "path"